src/joystick/SDL_joystick.c

```diff
@@ -116,7 +116,7 @@ static SDL_bool SDL_joysticks_initialized;
 static SDL_bool SDL_joysticks_quitting;
 static SDL_bool SDL_joystick_being_added;
 static SDL_Joystick *SDL_joysticks SDL_GUARDED_BY(SDL_joystick_lock) = NULL;
-static SDL_AtomicInt SDL_last_joystick_instance_id SDL_GUARDED_BY(SDL_joystick_lock);
```

Comment:
Just making sure this didn't land by accident (it's unrelated, and...is this not guarded by a lock anymore?)

Reply:
I'll remove that. I do want to check it in, but not as part of this commit. The annotation is wrong, that variable isn't actually guarded by the lock.